src/provider/view/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import {ClassNameBuilder} from '../../utils/ClassNameBuilder';
+
+import * as Styles from './Icon.module.scss';
+
+export enum IconState {
+    DEFAULT = 'default',
+    DISABLED = 'disabled',
+    ACTIVE = 'active'
+}
+
+interface Props {
```

Comment:
Can you remove the unused props? Things like `forceHeight`, `scale` sound like things that should be enforced rather than configurable - and not seeing any component that uses these properties. Probably others too.